src/easylink/rule.py

```diff
@@ -139,9 +139,9 @@ def _build_shell_command(self) -> str:
         '''
         export DUMMY_CONTAINER_OUTPUT_PATHS={",".join(self.output)}
         export DUMMY_CONTAINER_DIAGNOSTICS_DIRECTORY={self.diagnostics_dir}"""
-        for slot_name, slot_files in self.input_slots.items():
+        for slot_attrs in self.input_slots.values():
             shell_cmd += f"""
-        export {slot_name}={",".join(slot_files)}"""
+        export {slot_attrs["env_var"]}={",".join(slot_attrs["filepaths"])}"""
```

Comment:
Definitely seems like an improvement to have an explicit `"env_var"` here instead of `slot_name`